sxhkd-mode.el

```diff
@@ -108,8 +111,8 @@ Can be t, nil or ask"
       (if (looking-at-p (rx (or "#" (and bol eol))))
           0
         (end-of-line)
-        (forward-char -1)
-        (if (looking-at-p (rx "\\")) 0 1)))))
+        (forward-char (* -1 sxhkd-mode-indentation-length))
```

Comment:
Could you please change this line to `(- sxhkd-mode-indentation-length)`? Thanks!

Reply:
Thanks for the review, it is fixed in last commit.